deis-dev/tpl/deis-objectstorage-secret.yaml

```diff
@@ -8,7 +8,9 @@ metadata:
   annotations:
     deis.io/objectstorage: "{{.storage}}"
 type: Opaque
-data: {{ if eq .storage "gcs" }}
+data: {{ if eq .storage "minio" }}
+  access-key-id: OFRaUlkySlJXTVBUNlVNWFI2STUK
```

Comment:
We should generate this in objectstorage.yaml then in the same way minio-user is generated instead of hard-coding the access key and secret for every installation.